ubtue_Open Journal Systems Multi-Wrapper.js

```diff
@@ -5,11 +5,11 @@
 	"target": "/issue/view/|index.php/journal/(?!article/view/)|index.php/eguzkilore/",
 	"minVersion": "3.0",
 	"maxVersion": "",
-	"priority": 99,
-	"inRepository": false,
+	"priority": 150,
```

Comment:
Seems a bit risky, because like this the vanilla OJS translator is prioritized higher than our custom ones. Maybe 98 for ubtue_Open Journal Systems Standard.js and 99 for this translator?

Reply:
We decided a while ago that this translator would not be used any more. For this cause, i increased its priority.

Reply:
Sorry, ich wolle nur wissen, was diese Funktion macht.